### ISSUE TITLE
Couldn't add member / signup

### ISSUE THREAD

**Issue by danieland** [kylegregory/EmmaSharp#17]
Hi Kyle, I tried using your API but I kept getting this error:

```
Server Error in '/' Application.
JSON integer 3693545479 is too large or small for an Int32. Path 'member_id', line 3, position 26.


Stack Trace: 
[JsonReaderException: JSON integer 3693545479 is too large or small for an Int32. Path 'member_id', line 3, position 26.]
   Newtonsoft.Json.JsonTextReader.ParseNumber() +1834
   Newtonsoft.Json.JsonTextReader.ParseValue() +1164
   Newtonsoft.Json.JsonReader.ReadAsInt32Internal() +51
   Newtonsoft.Json.Serialization.JsonSerializerInternalReader.ReadForType(JsonReader reader, JsonContract contract, Boolean hasConverter) +152
   Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject(Object newObject, JsonReader reader, JsonObjectContract contract, JsonProperty member, String id) +2194
   Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue) +881
   Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent) +750
   Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType) +281
   Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings) +168
   Newtonsoft.Json.JsonConvert.DeserializeObject(String value, JsonSerializerSettings settings) +61
   EmmaSharp.EmmaApi.Execute(RestRequest request, Int32 start, Int32 end) +590
```

For now, I'll revert to creating a WebClient and doing the call there, good luck with your API!

**Comment by kylegregory** [comment]
Hey @danieland,

Thanks for that. I think we're running into Member Ids longer than Int32 supports. Since that change may break some existing folks code, I've opened a branch and changed member_id to long (Int64). Could you give that a run and see if it helps with the longer ids?

https://github.com/kylegregory/EmmaSharp/tree/fix-member-ids

If that works, we'll commit it to the base code as a breaking change.

**Comment by danieland** [comment]
Hi Kyle, I'm sorry I don't have time to test it these days, but I'm sure
that will fix it, I've looked at the response from MyEmma and the member_id
is around 3xxxxxxxxx currently, 99% this will fix it!

tor. 10. mar. 2016 kl. 17.33 skrev Kyle Gregory <notifications@github.com>:

**Comment by kylegregory** [comment]
Fixed in commit f6b7d782ed3c5044f198e1b0a42fb9f770db4444
